x-pack/test/functional/apps/dashboard/group3/reporting/screenshots.ts

```diff
@@ -313,7 +313,7 @@ export default function ({
           updateBaselines
         );
 
-        expect(percentDiff).to.be.lessThan(0.09);
+        expect(percentDiff).to.be.lessThan(0.15);
```

Comment:
would this be needed if we re-generate the baseline image?

Reply:
How do you re-generate those?

Reply:
That's fine. We can tackle lowering the diff along with https://github.com/elastic/kibana/issues/138809

Reply:
The easy way to update it: use the temporary file saved from the test run. It's just a saved file ignored by git.